pipelines/live-1/main/tzcronjobber.yaml

```diff
@@ -0,0 +1,42 @@
+resources:
+- name: tools-image
+  type: docker-image
+  source:
+    repository: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/tools
+    tag: latest
+    aws_access_key_id: ((aws-live-1.access-key-id))
+    aws_secret_access_key: ((aws-live-1.secret-access-key))
+- name: cronjobber-image
+  type: docker-image
+  source:
+    repository: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/tzcronjobber
+    tag: 0.2.0
```

Comment:
I think we should not specify the version here, there's no reason why we need to point to a specific version, we're only building and pushing. Less places to update on new releases ;)